test/test_tcp.rs

```diff
@@ -8,10 +8,10 @@ use std::time::Duration;
 
 use net2::{self, TcpStreamExt};
 
+use crate::{TryRead, TryWrite};
```

Comment:
The imports here are separated by newline, while everywhere else they're squished together.